src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * 
+ * stores a list of tags added or removed in the session
+ *
+ */
+public class Tagging {
+	
+	private static final ArrayList<String> TAGGINGLIST = new ArrayList<String>();
```

Comment:
Based on the given diagram, the Tagging list is not the Tagging class. If it was, the diagram would be like this:
![image](https://cloud.githubusercontent.com/assets/1673303/18409539/0e57b8c4-777c-11e6-846b-33dc86154bf8.png)